src/herring/node.hpp

```diff
@@ -0,0 +1,64 @@
+#pragma once
```

Comment:
Copyright header

Reply:
After this second comment, decided to not repeat it but applies to all new code files